app/src/components/TaskBoard.js

```diff
@@ -1,14 +1,22 @@
 import React, { useEffect, useState, useCallback } from "react";
 
-import * as apiClient from "../apiClient";
+import * as dbRequest from "../dbRequest";
 
-function TaskBoard() {
-  const [tasksList, setTasksList] = useState("");
+import Task from "./Task.js";
+import TaskAddForm from "./TaskAddForm";
 
-  async function getTasksInfo() {
-    let data = JSON.stringify(await apiClient.getTasks());
-
-    setTasksList(data);
+function TaskBoard({
+  isAuthenticated,
+  gcal,
+  timedownAccount,
+  setDetails,
+  setAction,
+  setTasksList,
+  tasksList,
+  getTasksInfo,
+}) {
+  function taskAction(action) {
+    setAction(action);
```

Comment:
This function seems like it's not really pulling its weight here.  Can you get away with just calling `setAction` wherever `taskAction` is currently called?  I think that should work since I only see it used once inside the button's `onClick`, as the body of an anonymous function.